Godeps/Godeps.json

```diff
@@ -1,6 +1,6 @@
 {
 	"ImportPath": "github.com/docker/libnetwork",
-	"GoVersion": "go1.4.2",
+	"GoVersion": "go1.5.1",
```

Comment:
No, but it's version for Docker now, I think it makes sense to test libnetwork with same, because docker is sole user of libnetwork now.